src/Calculator/Controls/EquationTextBox.cs

```diff
@@ -0,0 +1,719 @@
+                if (m_colorChooserMenuItem != null)
+                {
+                    m_colorChooserMenuItem.Text = resProvider.GetResourceString("colorChooserMenuItem");
+                    m_colorChooserMenuItem.Click +=  OnColorChooserButtonClicked;
+                }
+
+                if (m_functionButton != null)
+                {
+                    m_functionButton.Click +=  OnFunctionButtonClicked;
+                    m_functionButton.IsEnabled = false;
+                }
+
+                if (m_kgfEquationMenuItem != null)
+                {
+                    m_kgfEquationMenuItem.Text = resProvider.GetResourceString("functionAnalysisMenuItem");
+                    m_kgfEquationMenuItem.Click +=  OnFunctionMenuButtonClicked;
+                }
+
+                if (ColorChooserFlyout != null)
+                {
+                    ColorChooserFlyout.Opened += OnColorFlyoutOpened;
+                    ColorChooserFlyout.Closed += OnColorFlyoutClosed;
+                }
+
+                if (m_cutMenuItem != null)
+                {
+                    m_cutMenuItem.Click +=  OnCutClicked;
+                }
+
+                if (m_copyMenuItem != null)
+                {
+                    m_copyMenuItem.Click +=  OnCopyClicked;
+                }
+
+                if (m_pasteMenuItem != null)
+                {
+                    m_pasteMenuItem.Click +=  OnPasteClicked;
+                }
+
+                if (m_undoMenuItem != null)
+                {
+                    m_undoMenuItem.Click +=  OnUndoClicked;
+                }
+
+                if (m_selectAllMenuItem != null)
+                {
+                    m_selectAllMenuItem.Click +=  OnSelectAllClicked;
+                }
+
+                UpdateCommonVisualState();
+                UpdateButtonsVisualState();
+
+            }
+
+            protected override void OnPointerEntered(PointerRoutedEventArgs e)
+            {
+
+                m_isPointerOver = true;
+                UpdateCommonVisualState();
+
+            }
+
+            protected override void OnPointerExited(PointerRoutedEventArgs e)
+            {
+
+                m_isPointerOver = false;
+                UpdateCommonVisualState();
+
+            }
+
+            protected override void OnPointerCanceled(PointerRoutedEventArgs e)
+            {
+
+                m_isPointerOver = false;
+                UpdateCommonVisualState();
+
+            }
+
+            protected override void OnPointerCaptureLost(PointerRoutedEventArgs e)
+            {
+
+                m_isPointerOver = false;
+                UpdateCommonVisualState();
+
+            }
+
+            private void OnIsAddEquationModePropertyChanged(bool oldValue, bool newValue)
+            {
+
+                UpdateCommonVisualState();
+                UpdateButtonsVisualState();
+
+            }
+
+            private void UpdateCommonVisualState()
+            {
+
+                String state = null;
```

Comment:
In this file, some places use String while some use string. Although they both represent the same thing, the reading experience in VS is different. Could you please change the String to string?

Reply:
Scanned the codebase. Replaced `String` with `string`. Done same replacement for `Object` as well.